views/layout.erb

```diff
@@ -0,0 +1,8 @@
+<STYLE>A {text-decoration: none;} </STYLE>
+<div class="topnav" style="text-align: right;">
+  <a class="button" href="/listings">Listings</a>
+  <a class="button" href="#contact">Contact</a>
+  <a class="button" href="#about">About</a>
```

Comment:
Could we please have a sign in button in the navigation?
The route would be a get to '/sessions/new'
Will take a look at what a sign-out button would entail (As discussed it's probably going to be a hidden DELETE request so we'll need to use a form element.